java/vector/src/main/codegen/templates/DenseUnionVector.java

```diff
@@ -36,6 +35,7 @@
 import org.apache.arrow.vector.types.pojo.ArrowType;
 import org.apache.arrow.vector.types.pojo.Field;
 import org.apache.arrow.vector.types.pojo.FieldType;
+import org.apache.arrow.vector.util.DataSizeRoundingUtil;
```

Comment:
This line should be placed after the next line, to maintain the import order

Reply:
True. I changed it.

Reply:
We need to import it only once

Reply:
Sorry about that. Must have happened during the rebasing of my local changes. It's finally fixed now.